preDeploy.sh

```diff
@@ -0,0 +1,36 @@
+#
+# (C) Copyright IBM Corp. 2021, 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -xe
+echo "pre deploy code goes here"
+
+echo "IDS_PROJECT_NAME=${IDS_PROJECT_NAME}"
+
+#templateBranch="master"
+#if echo $Build_Prefix | egrep "^(\S*)-SCD$";then
+#  curl -sSL -u "watkins0@us.ibm.com:${gitApiKey}" "https://raw.github.ibm.com/whc-toolchain/njss-ops-overrides/${templateBranch}/${CLUSTER_NAME}/override.yaml" > chart/nodejs-starter/override.yaml
+#elif echo $Build_Prefix | egrep "^(\S*)-SCVV$";then
+#  curl -sSL -u "watkins0@us.ibm.com:${gitApiKey}" "https://raw.github.ibm.com/whc-toolchain/njss-ops-overrides/${templateBranch}/${CLUSTER_NAME}/override.yaml" > chart/nodejs-starter/override.yaml
+#elif echo $Build_Prefix | egrep "^(\S*)-PCD$";then
+#  curl -sSL -u "watkins0@us.ibm.com:${gitApiKey}" "https://raw.github.ibm.com/whc-toolchain/njss-ops-overrides/${templateBranch}/${CLUSTER_NAME}/override.yaml" > chart/nodejs-starter/override.yaml
+#else
+#  curl -sSL -u "watkins0@us.ibm.com:${gitApiKey}" "https://raw.github.ibm.com/whc-toolchain/njss-dev-overrides/${templateBranch}/${CLUSTER_NAME}/override.yaml" > chart/nodejs-starter/override.yaml
+#fi
+#cat chart/nodejs-starter/override.yaml
+#ls -la chart/nodejs-starter/
+
+# get this value from KP-CDT-SAMPLE Keyprotect instance
+# export REPLCOUNT="1"
+
+if echo "$PIPELINE_TOOLCHAIN_ID" | egrep "430afb18-4418-4ce9-95ea-5980bbaa7760"; then
```

Comment:
Could you explain the significance of `430afb18-4418-4ce9-95ea-5980bbaa7760`?

Reply:
This file came from the whc-starter app which is used as a starting point for these toolchain projects. I had been advised to leave that in, although the person I reviewed it with didn't recall exactly the bug ticket, they thought it was related to a toolchain bug.